src/components/Modal/Modal.stories.tsx

```diff
@@ -26,13 +26,13 @@ function BasicModal() {
 }
 
 function FormModal() {
-  const { register, getValues, triggerValidation, errors } = useForm()
+  const { register, getValues, trigger, errors } = useForm()
 
   const onValidate = useCallback(async () => {
-    await triggerValidation()
+    await trigger()
```

Comment:
trigger a le même comportement que triggerValidation ?

Reply:
Je vais vérifier mais triggerValidation a été supprimé et replacé par trigger

Reply:
https://react-hook-form.com/migrate-v5-to-v6/

C'est juste un renommage